about/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Об авторе</title>
+</head>
+<body>
+    <h1>Почему я выбрал front-end</h1>
+    <p>Всё началось в не так далёком <strong>2018 году</strong> когда я начал служить в наших <em>доблестных пограничных войсках.</em><br>Тогда я как обычно сидел в <mark>youtube</mark> и случайно наткнулся на обучающее видео по HTML. Тогда я его просто просмотрел и немного стало интересно а как это работает.<br> После пробовал самостоятельно учить всё это но информации для изучения стало слишком много и я решил пойти куда-нибудь учиться.<br> Я купил курс на FructCode там показывали как делается макет сайта с фильмами. шло время, а я так толком ничему и не научился, плюс на работе было просто не до учёбы и я забил на это всё.</p>
+    <p>Прошлой зимой я уволился из ОПС и начал работать тренером по л/атлетике. З/п у тренера очень маленькая и я решил пойти учиться куда-нибуть чтоб в последующем устроиться на нормальную работу.<br> Вот тут и вспомнил про все то что учил когда-то и решил дальше двигаться в этом направлении и чем глубже я погружаюсь тем мне всё более интересно как это работает и как решать все новые более сложные задачи.</p>
+    <hr>
+    <h2>Мои качества разработчика</h2>
+    Я бы не сказал что я прям уж <ins>такой крутой разработчик</ins>, но, как мне кажется, у меня есть качества которые необходимы разработчику на данном, а именно: упорство, <del>желание познавать что-то новое</del> (развивать свои скилы), <sub>умение работать в команде</sub>, <sup>усидчивость</sup> и хорошая концентрация направленная на достижение результата. А остальные качества будут развиваться в процессе обучения и, надеюсь, в последующей работе.
+    <hr>
+    <blockquote>
+    <p>Всё, что ни происходит, всегда так, как нужно, и только к лучшему.</p>
+    <cite>Белая гвардия (Михаил Афанасьевич Булгаков)</cite>
+    </blockquote>
+    
+    <hr>
+    <h3>Ссылки</h3>
+    <a href="https://github.com/Stackeevich/if-html">My repository</a>
+    <a href="https://statskeevich@gmail.com">My mail</a>
```

Comment:
Better to use "mailto:" cause the current implementation for me personally opens my Gmail  and this can be confusing for an ordinary user